### PR TITLE
update

### PR DESCRIPTION
**Pull request by afsaneh-rezaeinejad-deriv**
after changes of [this card](https://app.clickup.com/t/20696747/WALL-3964) is merged, onbaording test is failing, cypress code should be updated based on new changes
CU card: https://app.clickup.com/t/20696747/WALL-4097

### CODE REVIEW

File: cypress/e2e/wip/Wallets/onboarding.cy.js [hunk 1]
```diff
@@ -1,116 +1,141 @@
...
+  description
+) => {
+  cy.findByRole('alertdialog').should('exist')
+  cy.findByText(title).should('exist')
+  cy.findByText(description).should('exist')
+  cy.findByRole('button', { name: 'Back' }).should(
+    backButtonShouldExist ? 'exist' : 'not.exist'
+  )
+  cy.findByRole('button', { name: nextButtonName }).should('exist').click()
+  cy.wait(800)
```
Comment (markodonnell-deriv) [inline cypress/e2e/wip/Wallets/onboarding.cy.js:90]: Is this definitely needed? :)

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]

**Review by meeyun-deriv — APPROVED** [review]
LGTM (other than the 800 as commented by Mark)

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Pull request merged**